CalculatorIdo/Script.js

```diff
@@ -0,0 +1,149 @@
+window.onload = () => {clearCalculator();}
+
+var currAns = '';
+var isResultShown = false;
+var isbracketOpen = false;
+
+clearCalculator = () => {document.calcForm.input.value = '';}
+
+del = () => {
+    if(isResultShown) {
+        isResultShown = false;
+        clearCalculator();
+    }
+
+    let currString = document.calcForm.input.value;
+
+    // Deleted the whole word if its a word
+    if(/^[a-zA-Z]+$/.test(currString.charAt(currString.length - 1))) {
+        document.calcForm.input.value = currString.substring(0, currString.length - 1);
+        currString = document.calcForm.input.value;
+
+        while((/^[a-zA-Z]+$/.test(currString.charAt(currString.length - 1)))) {
+            document.calcForm.input.value = currString.substring(0, currString.length - 1);
+            currString = document.calcForm.input.value;
+        }
+    } else {
+        document.calcForm.input.value = currString.substring(0, currString.length - 1);
+    }
+}
+
+insert = (input) => { 
+    if(isResultShown) {
+        isResultShown = false;
+        clearCalculator();
+    }
+
+    document.calcForm.input.value = document.calcForm.input.value + input;
+}
+
+equal = () => {
+    if(isResultShown) {
+        isResultShown = false;
+        clearCalculator();
```

Comment:
It seems that every time you clear the calculator you set isResultShown to false, maybe it should in that function?